### ISSUE TITLE
EINVAL: invalid argument, chown

### ISSUE THREAD

**Issue by michaelfoley1** [jspm/npm#169]
I recently attempted to run `jspm:upgrade` inside a docker container configured with [user namespace remapping](https://docs.docker.com/engine/security/userns-remap/#about-remapping-and-subordinate-user-and-group-ids) enabled.

It's worth noting that i didn't get this error when i wasn't inside the container.

I quickly found myself with the following error:
```bash
     Configuration file jspm.config.js does not exist, create it?: Yes
     Creating registry cache...
...
...
...
     Downloading npm:video.js@5.12.6

err  Error on download for npm:video.js
Error: EINVAL: invalid argument, chown '/root/.jspm/packages/npm/video.js@5.12.6/package.json'
```
Tracing this error back to origin i found that it originated here: mafintosh/tar-fs/blob/v1.16.0/index.js#L231
```javascript
chown(name, header.uid, header.gid, cb)
```
It was being called with these args:
```javascript
chown("/root/.jspm/packages/npm/video.js@5.12.6/package.json",
   56734284,
   1075358037, 
   function (err) {
       self._locked = false
       if (err) return self.destroy(err)
       if (!self._stream) oncontinue()
   }
);
```

here's my attempt at validating the error using the command line:
```bash
bash-4.3# chown 56734284:1075358037 /root/.jspm/packages/npm/video.js\@5.12.6/package.json 
chown: /root/.jspm/packages/npm/video.js@5.12.6/package.json: Invalid argument
bash-4.3#
```

**Comment by aluanhaddad** [comment]
Thanks for the bug report and for the fix!

**Comment by michaelfoley1** [comment]
No worries, thanks for responding so quickly.

Would it be possible to get a release tagged? I believe we're up to 0.30.4

**Comment by aluanhaddad** [comment]
Let us ask @guybedford.

**Comment by guybedford** [comment]
Sure, I've published a 0.30.4.
